src/function/signal/zpk2tf.js

```diff
@@ -0,0 +1,79 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'zpk2tf'
+
+const dependencies = [
+  'typed',
+  'add',
+  'multiply',
+  'Complex',
+  'number'
+]
+
+export const createZpk2tf = /* #__PURE__ */ factory(name, dependencies, ({ typed, add, multiply, Complex, number }) => {
+  /**
+     * Compute the transfer function of a zero-pole-gain model.
+     *
+     * Syntax:
+     *      math.zpk2tf(z, p, k)
+     *
+     * Examples:
+     *    math.zpk2tf([1, 2], [-1, -2], 1)    // returns [[1, -3, 2], [1, 3, 2]]
+     *
+     * See also:
+     *   freqz
+     * @param {Array} z
+     * @param {Array} p
+     * @param {number} k
+     * @return {Array}
+     *
+     */
+  return typed(name, {
+    'Array,Array,number': function (z, p, k) {
```

Comment:
Thanks for adding the Matrix signatures to `freqz`. Can you add those too for `zpk2tf`?